cf-runagent/cf-runagent.c

```diff
@@ -250,10 +255,13 @@ static GenericAgentConfig *CheckOpts(int argc, char **argv)
     GenericAgentConfig *config = GenericAgentConfigNewDefault(AGENT_TYPE_RUNAGENT, GetTTYInteractive());
 
     DEFINECLASSES[0] = '\0';
-    SENDCLASSES[0] = '\0';
+    SENDCLASSES[0]   = '\0';
+    REMOTEBUNDLES[0] = '\0';
 
-    while ((c = getopt_long(argc, argv, "t:q:db:vnKhIif:D:VSxo:s:MH:C::l",
-                            OPTIONS, NULL)) != -1)
+    int longopt_idx;
+    while ((c = getopt_long(argc, argv, "t:q:db:vnKhIif:D:VSxo:s:MH:C::lB:",
+                            OPTIONS, &longopt_idx))
+           != -1)
```

Comment:
You are not checking ``B`` in the switch statement below. Did you mean to remove ``-B`` after we discussed in the review? Then it should be removed from here as well, I suppose.